test/e2e/tests/test_elasticsearch_domain.py

```diff
@@ -152,7 +159,7 @@ def test_create_delete_7_9(self, es_client):
         wait_for_delete_or_die(es_client, resource, timeout)
 
 
-    def test_create_delete_2d3m_multi_az_no_vpc_7_9(self, es_client):
+    def test_create_delete_2d3m_multi_az_no_vpc_7_9(self, es_client, resources):
```

Comment:
where are you using the `resources` fixture in this method?

Reply:
will update